src/coreComponents/dataRepository/Group.hpp

```diff
@@ -1712,6 +1730,22 @@ Wrapper< T > & Group::registerWrapper( string const & name,
   return rval;
 }
 
+template< typename LOG_LEVEL_INFO >
+std::enable_if_t< logInfo::is_log_level_info< LOG_LEVEL_INFO >, void >
+Group::addLogLevel()
+{
+  Wrapper< integer > * wrapper = getWrapperPointer< integer >( viewKeyStruct::logLevelString() );
+  if( wrapper == nullptr )
+  {
+    wrapper = &registerWrapper( viewKeyStruct::logLevelString(), &m_logLevel );
```

Comment:
I suppose this implicitly adds a `logLevel` wrapper to the `Group`. Would there be a danger of calling this outside a `Group` constructor and changing the schema?

Reply:
Good point, all `m_logLevelsRegistry ` is set  to nullptr  in `postInputInitializationRecursive ` so  I've added a `GEOS_ERROR_IF` in `addLogLevel` to check if `m_logLevelsRegistry `is set to `nullptr`

Reply:
@dkachuma 
I indeed adds the `logLevel` wrapper, as `enableLogLevel()` did before this PR.
Calling it during or after the `postInputInitialization` phase will generate an error as the `LogLevelRegistry` are released (`Group.cpp:248`).

Your point applies to all `Wrapper` descriptions, we may generalize this solution for all of them later.

Reply:
@MelReyCG, yes I think that would be better. A compile time check would be preferable IMO but it's not obvious to me how we could do it.

Reply:
Just an idea: we could do that by adding a general wrapper & group `GeneratedDocsRegistry` that would only be created & used during `ProblemManager::generateDocumentation()`.
It could have a generic `GenerateDescription< GROUP_T >()` method that could be implemented for each `Group` types (in their respective CPP files).
This way we could free the documentation generation & storage from the simulation runs, and the documentation would be stored at only one place rather than in each `Group` instance.